swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        
+        self.inventory = inventory if inventory is not None else []
+
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
Nice check to make sure the value is there before we try to remove it. Note that we could instead use try/except to handle the case where `remove` raises an error when the value isn't found.

As written, I'd suggest doing the check for the value _not_ being here first, and then returning as a guard clause, as this allows the main focus of the code to be less indented.

```python
    def remove(self, item):
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```